astropy/stats/funcs.py

```diff
@@ -729,6 +733,11 @@ def median_absolute_deviation(a, axis=None):
     axis : int, optional
         Axis along which the MADs are computed.  The default (`None`) is
         to compute the MAD of the flattened array.
+    ignore_nan : bool
+        Ignore NaN values (treat them as if they are not in the array) when
+        computing the median.  This will use `numpy.nanmedian`, which requires
```

Comment:
The docstring should be updated that it uses `np.ma.median` if `nanmedian` is not avaible.